src/test/java/VariablesTest.java

```diff
@@ -0,0 +1,39 @@
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import java.io.IOException;
+
+/**
+ * This test class is meant to test that our Sb2 program can correctly count
+ * the number of global variables, script variables, and sprite variables.
+ * @version 1
+ * @author James Ward
+ */
+
+public class VariablesTest {
+    /**
+     * Test that the Sb2 can accurately count the number of global variables.
+     */
+    @Test
+    public void testGlobalVariablesCount() {
+        Sb2 scattariaSb2 = new Sb2(Utils.getScratchariaJSONObject());
+        int expectedGlobVar = scattariaSb2.getGlobalVariableCount();
+        int actual = 23;
```

Comment:
Here and below you've got actual and expected backwards.  The actual count that our code comes up with is `scattariaSb2.getGlobalVariableCount()`, and the count that we expect is 23.